### ISSUE TITLE
Generated PKPASS File not executing on the iPhone

### ISSUE THREAD

**Issue by iamadilmughal** [alexandercerutti/passkit-generator#120]
## Running OS

Windows 11 
Node Version 16.6.0

## Description

I am using the following code to generate a PKPASS File in Node. 

```
  const pass = await PKPass.from({
    model: "./template/examplePass.pass",
    certificates: {
      wwdr,
      signerCert,
      signerKey,
      signerKeyPassphrase
    },
  });

  const buffer = pass.getAsBuffer();

  fs.writeFileSync("check.pkpass", buffer);

```

The ```examplePass.pass``` is the same pass template that we have in the ```/examples/models``` directory (after changing the values for ```passTypeIdentifier``` and ```teamIdentifier``` of course :) )

What happens is that the file gets generated successfully. But when I try to add it to apple wallet, it does not get added. When checked on Mac Console application, it gives the error 

```
Invalid data error reading pass pass.com.xxxxx.xxxxxxxxxxxxx/****. Manifest signature did not verify successfully
```

Apart from apple wallet, this file gets executed by all other applications (on android, on pc, etc.). 

## Expected behavior

The generated file should run on the apple wallet. 

## Steps to reproduce

- Download the examplePass.pass from ```/examples/models``` directory
- Change the values in ```pass.json``` according to your certificates
- Copy the code above and run it 

## Were you able to verify it by using (and changing) the examples?
Yes, I did. I used different templates to generate the pkpass but none worked. 

## Other details

- [x] I'm available to open a Pull Request to resolve the problem (after the triage)

**Comment by alexandercerutti** [comment]
Hey there, @iamadilmughal, thanks for using passkit-generator!

Do you have the chance to test your code and certificates on a MacOS or Linux machine?
I don't have a Windows machine to test on, so I'd like to ensure this is not a problem that is happening with Windows.

Also, did you cross-checked your certificates data `Subject` as per [this paragraph](https://github.com/alexandercerutti/passkit-generator/wiki/Generating-Certificates#which-are-the-passtypeidentifier-and-teamidentifier-am-i-using)?

I tried to remap the sha-1 of the files of a generated pass you sent me and checked them with sha-1s inside the `manifest.json`, and they match correctly.

Let me know,
Alexander

**Comment by iamadilmughal** [comment]
Hey @alexandercerutti, thanks for the reply. 

- No, I only tried to generate it using Windows OS. I will try to generate it using Linux and will let you know the output. 
- Yes, I did cross-verify the pass.json data with the Subject property of the certificate and the values were matching.
- If the hashing is matching in the manfiest.json, then what do you think is causing the error? Code and pass template are really basic level so they should not have any issues at all...

**Comment by iamadilmughal** [comment]
Hey @alexandercerutti, I did try and generate the pkpass file from Mac OS but have the same results.

**Comment by alexandercerutti** [comment]
Hey @iamadilmughal, I tried to read the content of the signature of the generated pass that you provided me via PM through this command, but I honestly didn't find anything wrong with a signature of a pass I generated, which works 🤔.

```sh
$ openssl cms -in signature -noout -cmsout -print -inform DER
```

I'm not quite sure. There must be something wrong with your certificates, but I'm unsure what to look for. If you share them with me privately, I might try to understand if there's something wrong.

I also tried to unpack the pkpass you provided me, remove signature and manifest, update the `passTypeIdentifier` and `teamIdentifier` and generate one by using that folder as a model to ensure there was nothing left behind in the model, and in fact **I was able to generate a working pass correctly**.
